Source/Core/Core/IOS/ES/ES.h

```diff
@@ -202,6 +202,9 @@ class ES : public Device
   u64 m_TitleID = -1;
   u32 m_AccessIdentID = 0x6000000;
 
+  static constexpr u8 MAX_OPEN_COUNT = 3;
```

Comment:
This can just be placed in the cpp file without the static specifier, since its not bound to the class in any way.